internal/cmd/integration-tests/main.go

```diff
@@ -30,26 +31,37 @@ func main() {
 
 func runIntegrationTests(cmd *cobra.Command, args []string) {
 	defer reportResults()
-	defer cleanUpEnvironment()
+
+	testFolder := "./tests/"
+	alloyBinaryPath := "../../../../../build/alloy"
+	alloyBinary := "build/alloy"
+
+	if runtime.GOOS == "windows" {
+		testFolder = "./tests-windows/"
+		alloyBinaryPath = "..\\..\\..\\..\\..\\build\\alloy.exe"
+		alloyBinary = "build/alloy.exe"
+	} else {
+		setupEnvironment()
+		defer cleanUpEnvironment()
```

Comment:
Why no setup or clean up on windows?

Reply:
can't use the linux images in the windows runner so we are quite limited in terms of environment. For now not using docker and only running one Alloy to run the windows exporter seems to be sufficient. I can rename the function to "setupDockerEnv" and "cleanupDockerEnv" to make it more clear

Reply:
Not sure that it would be really worth it because besides the docker part which is quite small, they would re-use everything. Also I don't think that we need a linux standalone. And maybe one day the windows runner will be able to use WSL 2 and it will work with Mimir linux image or at some point we may want to run a docker windows image and the difference will be gone.
I agree that now it looks a bit convoluted though. I can structure the test better so that it looks less chaotic